tests/integration/shared/sources/new-or-changed-sources-test.js

```diff
@@ -156,12 +156,18 @@ test('Teardown', async t => {
    * listening to the sandbox port + 1 (see
    * https://github.com/architect/sandbox/blob/master/src/sandbox/index.js,
    * search for 'process.env.ARC_EVENTS_PORT').  If the sandbox is
```

Comment:
From here on out deals with a new exception during test teardown, has nothing to do with this draft code.